pkg/cluster/internal/create/create.go

```diff
@@ -53,7 +53,7 @@ const (
 // and suffix this name, we can relax it a little
 // see NewContext() for usage
 // https://godoc.org/github.com/docker/docker/daemon/names#pkg-constants
-var validNameRE = regexp.MustCompile(`^[a-zA-Z0-9_.-]+$`)
+var validNameRE = regexp.MustCompile(`^[a-z0-9_.-]+$`)
```

Comment:
this regex is going to change further, but we want to get that particular bug corrected already ...

Reply:
there is also a TODO to move this check to the config validation, is that the plan?

Reply:
yeah, it'll work here for now but